site/en/docs/crux/release-notes/index.md

```diff
@@ -43,6 +43,41 @@ The CrUX dataset on BigQuery is generally updated on the second Tuesday of every
 
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.
 
+## 202209
+
+[Announcement](TBC)
```

Comment:
To be updated with announcement link

Reply:
TODO

Reply:
Updated in https://github.com/GoogleChrome/developer.chrome.com/pull/3978/commits/22cefc9b396d9d53538dab32120468787181411b